### DISCUSSION TITLE
Office hours / Podcasts / Videos / Blogs?

### DISCUSSION THREAD

**Discussion by adrianhall** [CommunityToolkit/Datasync#119] [Ideas]
Hey folks,

I'm looking at potentially doing some developer advocacy work for the Datasync Community Toolkit, but I need your feedback.

1.  For live events, what time zones should I target?  Do you prefer early, during your day, or evening?  (Note: I am in Redmond which is US/Pacific, or UTC+8 and it's unlikely I will be doing things at the weekend or during my sleeping hours - trust me, I'm cranky when I don't get enough sleep).

2. How would you prefer to get information?  Some ideas:

  * A teams meeting where I cover a broad topic (advertised in advance) and you can do Q&A
  * Blog posts (ok - these are a given - I'm already working on them)
  * Videos
  * A podcast
  * Conferences / meetups (even virtual meetups)

3. What topics are top of mind for you (so I can cover those first)?

**Comment by myokeeh** [comment]
When I first got into using Azure Mobile Service, I found blog posts and videos (then Channel 9) useful. Being a guest on one of the mainstream C#/.NET podcasts is probably good enough. Having a podcast sounds interesting, but maybe too much to maintain yourself?

**Reply by adrianhall** [reply]
I'm intending on writing a bunch of blogs over the next few weeks, so stay tuned on this!

**Comment by hansmbakker** [comment]
The [book you wrote](https://adrianhall.github.io/develop-mobile-apps-with-csharp-and-azure/) about azure mobile apps was very useful, because it covered things like [conflict handling](https://adrianhall.github.io/develop-mobile-apps-with-csharp-and-azure/chapter3/client/#handling-conflict-resolution) in more detail than the documentation.

**Reply by adrianhall** [reply]
I suspect I would just go into more detail in the documentation now - a book is a LOT of effort and it didn't stay current for very long.

**Reply by hansmbakker** [reply]
good documentation is even better 👍🏼

**Reply by James-McMenamin** [reply]
Best practices NEVER die. eg "Gang of 4"

**Reply by adrianhall** [reply]
Can you expand on what you would expect to see, @James-McMenamin

**Reply by James-McMenamin** [reply]
Oh @adrianhall nothing nothing most of what is in that book I would 100% agree with and consider alot of it as Best practices.

**Comment by NicolasKlaas** [comment]
Bit late to the show, but for me its namely blogs and video(s) (YouTube?).

**Comment by atexinspect** [comment]
An updated version of this for me please - Syncing data with Azure Mobile Apps and .NET MAUI - https://www.youtube.com/watch?v=7ZcchuKWODQ

**Reply by adrianhall** [reply]
Oh god - I forgot I made that!  (Full disclosure, I *hate* being on camera).

Let me see if I can con one of the .NET dev evangelists to be on camera and post something.
